zeppelin-zengine/src/test/java/org/apache/zeppelin/interpreter/InterpreterFactoryTest.java

```diff
@@ -17,481 +17,50 @@
-    factory = new InterpreterFactory(conf, null, null, null, depResolver, true, interpreterSettingManager);
-    final InterpreterInfo info1 = new InterpreterInfo("className1", "name1", true, null);
-    interpreterSettingManager.add("group1", new ArrayList<InterpreterInfo>(){{
-      add(info1);
-    }}, new ArrayList<Dependency>(), new InterpreterOption(true), Collections.EMPTY_MAP, "/path1", null);
-
-    InterpreterOption perUserInterpreterOption = new InterpreterOption(true, InterpreterOption.ISOLATED, InterpreterOption.SHARED);
-    final InterpreterSetting setting1 = interpreterSettingManager.createNewSetting("test-group1", "group1", new ArrayList<Dependency>(), perUserInterpreterOption, new HashMap<String, InterpreterProperty>());
-
-    interpreterSettingManager.setInterpreters("user1", "note", new ArrayList<String>() {{
-      add(setting1.getId());
-    }});
-
-    interpreterSettingManager.setInterpreters("user2", "note", new ArrayList<String>() {{
-      add(setting1.getId());
-    }});
-
-    assertNotEquals(factory.getInterpreter("user1", "note", "test-group1"), factory.getInterpreter("user2", "note", "test-group1"));
-  }
-
-
-  @Test
-  public void testInvalidInterpreterSettingName() {
-    try {
-      interpreterSettingManager.createNewSetting("new.mock1", "mock1", new LinkedList<Dependency>(), new InterpreterOption(false), new HashMap<String, InterpreterProperty>());
-      fail("expect fail because of invalid InterpreterSetting Name");
-    } catch (IOException e) {
-      assertEquals("'.' is invalid for InterpreterSetting name.", e.getMessage());
-    }
-  }
-
-
-  @Test
-  public void getEditorSetting() throws IOException, RepositoryException, SchedulerException {
-    List<String> intpIds = new ArrayList<>();
-    for(InterpreterSetting intpSetting: interpreterSettingManager.get()) {
-      if (intpSetting.getName().startsWith("mock1")) {
-        intpIds.add(intpSetting.getId());
-      }
-    }
-    Note note = notebook.createNote(intpIds, new AuthenticationInfo("anonymous"));
-
-    Interpreter interpreter = factory.getInterpreter("user1", note.getId(), "mock11");
-    // get editor setting from interpreter-setting.json
-    Map<String, Object> editor = interpreterSettingManager.getEditorSetting(interpreter, "user1", note.getId(), "mock11");
-    assertEquals("java", editor.get("language"));
-
-    // when interpreter is not loaded via interpreter-setting.json
-    // or editor setting doesn't exit
-    editor = interpreterSettingManager.getEditorSetting(factory.getInterpreter("user1", note.getId(), "mock1"),"user1", note.getId(), "mock1");
-    assertEquals(null, editor.get("language"));
-
-    // when interpreter is not bound to note
-    editor = interpreterSettingManager.getEditorSetting(factory.getInterpreter("user1", note.getId(), "mock11"),"user1", note.getId(), "mock2");
-    assertEquals("text", editor.get("language"));
-  }
-
-  @Test
-  public void registerCustomInterpreterRunner() throws IOException {
```

Comment:
Do we have an equivalent test for this?

Reply:
It is covered here. https://github.com/apache/zeppelin/pull/2554/files#diff-6e873dfc1f240d8453d17a98f2ae4d10R69